packages/databricks-sdk-js/src/apis/workspace/model.ts

```diff
@@ -187,7 +187,13 @@ export interface DeleteSecret {
     scope: string;
 }
 
-export type ExportFormat = "DBC" | "HTML" | "JUPYTER" | "R_MARKDOWN" | "SOURCE";
+export type ExportFormat =
+    | "DBC"
+    | "HTML"
+    | "JUPYTER"
+    | "R_MARKDOWN"
+    | "SOURCE"
+    | "AUTO";
```

Comment:
@kartikgupta-db we need to make this change upstream. Do you have context why we need `AUTO`?

Reply:
AUTO allows us to upload both a notebook and a file from a common interface. We do not have to parse #Databricks notebook source comment. The API figures it out.

Reply:
This is actually for exporting. `import` does have the `AUTO` flag but `export` doesn't. From what I can see `export` is only being used in integ tests.